bundles/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/handler/CBusLightHandler.java

```diff
@@ -0,0 +1,132 @@
+ */
+@NonNullByDefault
+public class CBusLightHandler extends CBusGroupHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(CBusLightHandler.class);
+
+    public CBusLightHandler(Thing thing) {
+        super(thing, CBusBindingConstants.CBUS_APPLICATION_LIGHTING);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        Group group = this.group;
+        if (group == null) {
+            return;
+        }
+        if (command instanceof RefreshType) {
+            try {
+                int level = group.getLevel();
+                logger.debug("handle RefreshType Command for Chanell {} Group {} level {}", channelUID, groupId, level);
+                if (channelUID.getId().equals(CBusBindingConstants.CHANNEL_STATE)) {
+                    updateState(channelUID, (level > 0) ? OnOffType.ON : OnOffType.OFF);
+                } else if (channelUID.getId().equals(CBusBindingConstants.CHANNEL_LEVEL)) {
+                    updateState(channelUID, new PercentType((int) (level * 100 / 255.0)));
+                }
+            } catch (CGateException e) {
+                logger.debug("Failed to getLevel for group {}", groupId, e);
+            }
+        } else {
+            if (channelUID.getId().equals(CBusBindingConstants.CHANNEL_STATE)) {
+                logger.debug("Channel State command for {}: {}", channelUID, command);
+                if (command instanceof OnOffType) {
+                    try {
+                        if (command.equals(OnOffType.ON)) {
+                            group.on();
+                        } else if (command.equals(OnOffType.OFF)) {
+                            group.off();
+                        }
+                    } catch (CGateException e) {
+                        logger.debug("Failed to send command {} to {}", command, group, e);
+                    }
+                }
+            } else if (channelUID.getId().equals(CBusBindingConstants.CHANNEL_LEVEL)) {
+                logger.debug("Channel Level command for {}: {}", channelUID, command);
+                try {
+                    if (command instanceof OnOffType) {
+                        if (command.equals(OnOffType.ON)) {
+                            group.on();
+                        } else if (command.equals(OnOffType.OFF)) {
+                            group.off();
+                        }
+                    } else if (command instanceof PercentType) {
+                        PercentType value = (PercentType) command;
+                        group.ramp((int) Math.round(value.doubleValue() / 100 * 255), 0);
+                    } else if (command instanceof IncreaseDecreaseType) {
+                        logger.debug("Increase/Decrease not implemented for {}", channelUID);
+                    }
+                } catch (CGateException e) {
+                    logger.warn("Failed to send command {} to {}", command, group, e);
```

Comment:
If this is a communication error should it change the thing status to offline?

Reply:
Now set to offline.